### PR TITLE
Updates org info

### PR DESCRIPTION
**Pull request by sventi555**
uniappca -> yoiksoft

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@uniappca/type-validation-nodejs",
+  "name": "@yoiksoftca/type-validation-nodejs",
```
Comment (matootie) [inline package.json]: No need for "ca" as our GitHub org is @yoiksoft

```suggestion
  "name": "@yoiksoft/type-validation-nodejs",
```

File: package.json [hunk 2]
```diff
@@ -9,14 +9,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/uniappca/type-validation-nodejs.git"
+    "url": "git+https://github.com/yoiksoftca/type-validation-nodejs.git"
```
Comment (matootie) [inline package.json]: Same as above

```suggestion
    "url": "git+https://github.com/yoiksoft/type-validation-nodejs.git"
```

File: package.json [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
   },
   "author": "Stephen Edwards",
   "license": "UNLICENSED",
   "bugs": {
-    "url": "https://github.com/uniappca/type-validation-nodejs/issues"
+    "url": "https://github.com/yoiksoftca/type-validation-nodejs/issues"
```
Comment (matootie) [inline package.json]: Same as above

```suggestion
    "url": "https://github.com/yoiksoft/type-validation-nodejs/issues"
```

File: package.json [hunk 4]
```diff
@@ -9,14 +9,14 @@
...
   },
-  "homepage": "https://github.com/uniappca/type-validation-nodejs#readme",
+  "homepage": "https://github.com/yoiksoftca/type-validation-nodejs#readme",
```
Comment (matootie) [inline package.json]: Same as above

```suggestion
  "homepage": "https://github.com/yoiksoft/type-validation-nodejs#readme",
```

**Comment by matootie** [comment]
You can batch those change requests as one commit in the [files tab](https://github.com/yoiksoft/type-validation-nodejs/pull/3/files).

**Comment by sventi555** [comment]
it's always the smallest pull requests that are the hardest to nail on the first try...

**Pull request merged**